### ISSUE TITLE
Get Safety Score endpoints data

### ISSUE THREAD

**Issue by rog0978** [tdorssers/TeslaPy#37]
Anyone managed to get Safety Score data? I see the end points in endpoints.json, but they don't seem to work with e.g. the cli.py example.

**Comment by philipsd6** [comment]
FYI, you need to add `timezone` to the params, otherwise you get a validation error.

```python
...
    print(tesla.get(url, params={'vin': vehicles[0]['vin'],
                                 'deviceLanguage': 'en',
                                 'deviceCountry': 'US',
                                 'timezone': 'UTC'}))
```

**Comment by tdorssers** [comment]
Thanks, I have added this example to the readme!

**Comment by philipsd6** [comment]
yw! Now the only thing I'm struggling to figure out is what the numbers mean in the `dailyAggregation.metrics`. Seems like the Deciles are a float between 0 and 1, and they are opposite of what I understand decile rankings to be, and I guess it appears to be a ranking against all other drivers? The Median... the median of what? Same for Average -- average of what? Are these numbers aggregated by an unknown number of measurements taken during that daily period? I dunno if I did it right, but I made some graphs out of the data -- but if anyone has a better sense of the data, maybe that can be shared!? :)
